sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -9393,4 +9393,34 @@ ALTER TABLE testreport_history ADD COLUMN preliminary boolean;
 
 INSERT INTO schema_version (version_number, comment) VALUES (431, '[DEMIS2SORMAS] Handle New Profile: Preliminary Test Results #5551');
 
+-- 2021-12-03 [S2S] Implement outgoing S2S entity validation #7070
+ALTER TABLE areas ADD COLUMN centrally_managed boolean DEFAULT false;
+ALTER TABLE areas_history ADD COLUMN centrally_managed boolean DEFAULT false;
+
+ALTER TABLE community ADD COLUMN centrally_managed boolean DEFAULT false;
+/*ALTER TABLE community_history ADD COLUMN centrally_managed boolean DEFAULT false;*/
```

Comment:
No need to comment this here since the missing tables will most likely be added in a different schema version

Reply:
fixed